Woodoku/tests/Entity/test_woodoku_board.py

```diff
@@ -0,0 +1,121 @@
+from typing import Tuple
+
+import pytest
+
+from Exceptions.Exceptions import ShapeOutOfBoardError
+from WoodokuBoard import _WoodokuBoardRepresentation
+import numpy as np
+
+N = 9
+
+
+class TestWoodokuRepresentation:
+    one_block_lst = [(0, 0)]
+    three_block_lst = [(1, 3), (5, 8), (0, 7)]
+    five_block_lst = three_block_lst + [(2, 1), (0, 7)]
+
+    def board_after_adding(self, rep: _WoodokuBoardRepresentation, lst: list):
+        rep.add_blocks(lst)
+        return rep.board()
+
+    def board_after_removing(self, rep: _WoodokuBoardRepresentation, lst: list):
+        rep.remove_blocks(lst)
+        return rep.board()
```

Comment:
These are set up methods for a test
refer to this guide https://docs.pytest.org/en/6.2.x/xunit_setup.html or just search pytest setup or teardown 

I'm not sure about teardown, but perhaps investigate if any variables maybe persistance and not go out of scope during a test. any lingering variables should be deleted (I think either set to null or use the `del` keyword)

Reply:
I checked the same doc yesterday, but I found that `board_after_adding` and `board_after_removing` are helper functions instead of the code necessary to run before each test. My initial intention is that since both methods try to get access to the private field in the class, I would rather create a wrapper function for the getter and call it instead of calling getters anywhere in tests.

Reply:
First, the feature code is what is important. so feature code should not be altered for the convenience of the test code. 

Second, read the guide, there are methods setups/teardown that run once depending on the scenario. It could be once for the whole test file, once for the class, once for the class method or once for a function.